.github/workflows/welcome.yml

```diff
@@ -0,0 +1,19 @@
+name: Welcome
+on:
+ pull_request:
+  types: [opened, closed]
+ issues:
+  types: [opened]
+jobs:
+ run:
+  runs-on: ubuntu-latest
+  steps:
+   - uses: wow-actions/welcome@v1
+     with:
+      FIRST_ISSUE: |
+       👋 @{{ author }}
+       Welcome to SwiftWave Community! Star ⭐ the main repository [swiftwave](https://github.com/swiftwave-org/swiftwave) to grow this project and join our Slack community [invite link](https://join.slack.com/t/swiftwave-team/shared_invite/zt-21n86aslx-aAvBi3hv1GigVA_XoXiu4Q) to stay updated. Be sure to follow the issue template!
```

Comment:
We can remove the line `Be sure to follow the issue template!` for now !

Reply:
Okay, Doing necessary changes right away!